examples/ags/benchmark/gsm8k.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+# @Date    : 
+# @Author  : issac
+# @Desc    : test on gsm8k
+
+import json
+import re
+import os
+
+# 读取原始数据集
+def read_jsonl(path: str):
+    with open(path) as fh:
+        return [json.loads(line) for line in fh.readlines() if line]
+
+# 和图/和基础模型直接交互得到答案
+def LLM(question):
+    answer = ""
+    # 这里就是输入问题question返回答案answer
+    # answer = 根据question生成的回答
+    return answer
+
+def gsm_extract_answer(completion):
```

Comment:
seems benchmark like deepeval do the same thing. https://github.com/confident-ai/deepeval/blob/main/deepeval/benchmarks/gsm8k/template.py#L54

Reply:
@isaacJinyu

Reply:
yep, I have use it yesterday, But their solution was also a bit rough, so I made some optimizations and will submit it today.